libraries/AP_Airspeed/Airspeed_Calibration.cpp

```diff
@@ -146,11 +145,11 @@ void AP_Airspeed::update_calibration(uint8_t i, const Vector3f &vground, int16_t
             param[i].ratio.save();
             state[i].last_saved_ratio = param[i].ratio;
             state[i].counter = 0;
+            GCS_SEND_TEXT(MAV_SEVERITY_INFO, "AS %u ratio reset: %f", i , static_cast<double> (param[i].ratio));
```

Comment:
Let's not make peopls struggle with what AS is, and just print "Airspeed" the same as the rest of the library. I'm also dubious of the value of send texting it here.

Reply:
done